examples/ensemble/plot_stack_predictors.py

```diff
@@ -253,7 +261,7 @@ def plot_regression_results(ax, y_true, y_pred, title, scores, elapsed_time):
     )
     elapsed_time = time.time() - start_time
 
-    y_pred = cross_val_predict(est, X, y, n_jobs=2, verbose=0)
+    y_pred = cross_val_predict(est, X, y, n_jobs=2, verbose=0, cv=2)
```

Comment:
I feel like the `cv` setting should be consistent with `cross_validate`. Maybe @glemaitre has some input on this?

Reply:
yep. let's do that.

Reply:
I addressed this, using consistently 5-fold cv for cross_val_predict and cross_validate.

Reply:
Comparing the example on [dev](https://scikit-learn.org/dev/auto_examples/ensemble/plot_stack_predictors.html) and this [PR](https://169392-843222-gh.circle-artifacts.com/0/doc/auto_examples/ensemble/plot_stack_predictors.html) there does not look to be a timing difference anymore when we use the default `cv=5`.

Reply:
Yes, I confirm that most performance gain is lost once we put the `cv=5`'s back in place.